### DISCUSSION TITLE
Release inflolists as separate package for v2

### DISCUSSION THREAD

**Discussion by msnegurski** [filamentphp/filament#5609] [General]
Hey, from what I understand Livewire v3 will not be released soon, looks like it's months ahead at best. Infolists is very very desired feature, so would be cool if you could release it earlier as separate package if thats not a lot of work.

**Comment by danharrin** [comment]
While I understand the need for this completely, it is just too much work that we could avoid and spend time on more new stuff. I'm sorry

**Reply by msnegurski** [reply]
Completely undestandable, thanks for the answer and good work you are doing! 🙏
